src/app/code/community/Zendesk/Zendesk/Helper/Data.php

```diff
@@ -188,4 +268,230 @@ public function isExternalIdEnabled()
     {
         return Mage::getStoreConfig('zendesk/general/use_external_id');
     }
+    
+    /**
+     * Retrieve ticket statistics
+     */
+    public function getTicketTotals($type = null, $from = null, $to = null)
+    {
+        try {
+        $tickets = Mage::getModel('zendesk/api_tickets')->all();
+        }
+        catch(Exception $ex) {
+            return array();
+        }
+        
+        if( is_null($tickets) )
+        {
+            return false;
+        }
+        
+        $totals = array(
+            'open'      =>  0,
+            'new'       =>  0,
+            'solved'    =>  0,
```

Comment:
There is also an "on-hold" status which can be enabled for accounts. Is there a way these can be not hard-coded? Seem brittle.